custom-recipes/timeseries-forecast-1-train-evaluate/recipe.json

```diff
@@ -597,11 +598,19 @@
             "visibilityCondition": "false"
         },
         {
-            "name": "gpu",
+            "name": "use_gpu",
             "label": "Use GPU",
             "type": "BOOLEAN",
             "defaultValue": false,
-            "visibilityCondition": "'cpu' == 'gpu'"
+            "visibilityCondition": "'gpu' == 'gpu'"
```

Comment:
This will be changed by the make rule for gpu, can you leave the previous version? The tests will need to change this manually.